widget_driver/test/drivable_widget_test.dart

```diff
@@ -66,5 +67,90 @@ void main() {
 
       expect(find.text('didCallTestMethod: true'), findsOneWidget);
     });
+    testWidgets('On state change updateDriverProvidedProperties gets called', (WidgetTester tester) async {
```

Comment:
This test file gets a bit bigger.. should we devide it into different sections using more groups?
So having a structure something like this

```
main() {
    group('DrivableWidget:') {
        group('Runtime Environment:') { ... }

        group('Driver access:') { ... }

        group('Providable properties:') { ... }
    }
}
```